### DISCUSSION TITLE
How to run a program manually

### DISCUSSION THREAD

**Discussion by skarragallagher** [sebr/bhyve-home-assistant#158] [Q&A] [answered]
I have this unit https://www.orbitonline.com/products/b-hyve-smart-indoor-irrigation-controller

I have the integration running and it works well. I can manually turn on/off zones and see the history and last run etc.

I created a schedule and program with the App. I would like to have a button on the dashboard that would allow me to run the cycle. Is that possible? I was thinking maybe a helper binary switch for the trigger and then some call service in the automation but im not sure.

**Comment by sebr** [comment, marked as answer]
This isn't currently possible - you can follow issue #145 for updates - or please contribute a PR :) !!
